src/site/content/en/blog/vr-comes-to-the-web-pt-ii/index.md

```diff
@@ -0,0 +1,313 @@
+---
+title: Virtual reality comes to the web, part II
+subhead: All about the frame loop
+authors:
+  - joemedley
+date: 2020-02-12
+hero: hero.jpg
+alt: A person using a virtual reality headset.
+description:
+  Virtual reality came to the web in Chrome 79. Based on the WebXR Devicer API,
+  this launch is the foundation for both augmented and virtual reality. This
+  article is the second in a series, focusing on the frame loop, the part of
+  an XR session where images are shown to a viewer. Other browsers will soon be
+  supporting the WebXR Device API, including Firefox Reality, Oculus Browser,
+  Edge and Magic Leap's Helio browser, among others.
+tags:
+  - post
+  - augmented-reality
+  - virtual-reality
+  - webxr
+---
+
+Recently, I published [Virtual reality comes to the
+web](https://web.dev/vr-comes-to-the-web/), an article that introduced basic
+concepts behind the [WebXR Device
+API](https://wiki.developer.mozilla.org/en-US/docs/Web/API/WebXR_Device_API). I
+also provided instructions for requesting, entering, and ending an XR session.
+
+This article describes the frame loop, which is a user-agent controlled
+infinite loop in which content is repeatedly drawn to the screen. Content is
+drawn in discrete blocks called frames. The succession of frames creates the
+illusion of movement.
+
+## What this article is not
+
+WebGL and WebGL2 are the only means of rendering content during a frame loopß in
+a WebXR App. Fortunately many frameworks provide a layer of abstraction on top
+of WebGL and WebGL2. Such frameworks include [three.js](https://threejs.org/),
+[babylonjs](https://www.babylonjs.com/), and
+[PlayCanvas](https://playcanvas.com/), while [A-Frame](https://aframe.io/) and
+[React 360](https://facebook.github.io/react-360/) are designed for interacting
+with WebXR.
+
+This article is neither a WebGL nor a framework tutorial. It explains basics of
+a frame loop using the Immersive Web Working Group's Immersive VR Session sample
+([demo](https://immersive-web.github.io/webxr-samples/immersive-vr-session.html)
+[source](https://github.com/immersive-web/webxr-samples/blob/master/immersive-vr-session.html).
```

Comment:
Formatting is off here

![image](https://user-images.githubusercontent.com/4713486/74072282-2bc0f780-49bb-11ea-9c2a-c91d30e31911.png)